Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/RoomWrapper.java

```diff
@@ -71,11 +85,16 @@ public void join(PlayerWrapper player, String password) {
         String roomName = room.getName();
         LOGGER.log(Level.INFO, "Player {0} joined session {1}.", new String[]{player.getUsername(), roomName});
 
-        GameServerRmiController rmiController = GameServer.getInstance().getRmiControler();
+        GameServerRmiController rmiController = GameServer.getInstance().getRmiController();
         rmiController.increaseOccupancy(roomName);
         rmiController.changeHost(roomName, room.getHost().getUsername());
     }
 
+    /**
+     * Leaves the player of the Room.
+     *
+     * @param player The given player, not null.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1155)